modules/gathering-data-specific-features.adoc

```diff
@@ -110,6 +117,12 @@ ifndef::openshift-dedicated[]
 |Data collection for Local Storage Operator.
 endif::openshift-dedicated[]
 
+// TODO: Is this available on OKD / is this right?
```

Comment:
@dobsonj a few questions on the availability of the secret store CSI driver operator:

1. Is it available on OKD?
2. Is it available on dedicated?

Right now, I have it showing as yes to OKD and no to dedicated.

Reply:
It should be available in both, assuming both variants have OperatorHub.